### DISCUSSION TITLE
Plotting using HTML and Pywebview

### DISCUSSION THREAD

**Discussion by Sirut** [project-nv/night-vision#47] [Q&A]
How can i plot the chart using HTML as in Lightweigh-chart?
Actually im using pywebview for create a server and then calling the main html page for plot. Is possible doing the same with NightVision?

**Comment by C451** [comment]
You can try this: https://nightvision.dev/guide/intro/10-basic-examples.html#_9-using-the-lib-with-python

**Comment by Sirut** [comment]
Thanks for the answer C451. i tried the snippet you posted, but i get an error like that:
https://pastebin.com/ZEk11hD2

In _Lightweight-Chart_ i used to do something like that:
```
import webview
#webview.create_window('App', index.html', js_api=None, width=800, height=600)
#webview.start(debug=True,http_server=True)
```
Where in index.html was imported everything necessery to plot the data. Creating a server it's be usefull for avoid any problem with _CORS_. So this was enought for open a windows form with a webviewer, and plotting the chart.

Now, i tried to do the same thing with _NighVision_, but seem like he did not run on the server created by _pywebview_. Assuming that i created a 'data-1.json' file and imported in the 'main.js' for passing the OHCLV datas, so in this paritcular case i deleted the part of code in client.py where create a random datas.
Here a snippet of what i tried (everything else is like the example you posted from NighVision Docs):

**client.py**
```
webview.create_window('App', 'index.html', js_api=None, width=800, height=600, min_size=(200, 100))
webview.start(debug=True,http_server=True)
```

i dont get no error in this case, script open a form and load index.html, but it's completely blank, seems like script aren't running.

**Reply by C451** [reply]
I'm not a python coder, so you'll have to experiment with this. If you solve the problem, we can add your solution as an example.
